Sprint-2/debug/author.js

```diff
@@ -11,6 +12,6 @@ const author = {
   alive: true,
 };
 
-for (const value of author) {
+for (const value of Object.values(author)) {
   console.log(value);
```

Comment:
You can also use Object.entries(obj); 
but  while getting key value pairs you have to follow this pattern
```
const obj = { a: 5, b: 7, c: 9 };
for (const [key, value] of Object.entries(obj)) {

}
```